### PR TITLE
Rs 2022 05 12

### PR DESCRIPTION
**Pull request by rschlaikjer**
Only 2 bigass files remain

(in this route)

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 1]
```diff
@@ -71,38 +72,42 @@ I pour all my energy into that little finger.
...
+// Interpreting 感覚は頼りになる as like breaking through the wall, gaining a foothold
+// Second half rendered more as an internal monologue / psych up.
+Even if it's only my pinky, I've begun to claw back control over my body.
+ I can do this. I'm not just a doll. I can move as I please.
 }
 [sha:34b203eed0d7df7370475ab124dd553a2ab972d0]{
 -- Page 88, Offset 31925.
 -- 次いで、触覚がどんなものかを思い出すように、小指から薬指、手の平、肘、腕、肩、と動く場所を増やしていく。
-Then, as if to remind myself of what it's like to feel, I expand my range of movement from my pinky to my ring finger, palm, elbow, arm, and then my shoulder.
+Next, as though rediscovering my sense of proprioception,
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:109]: kinesthesia is a more common term imo

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 2]
```diff
@@ -112,17 +117,18 @@ Then, as if to remind myself of what it's like to feel, I expand my range of mov
...
 -- 痛みを感じなかったのは、全身が麻痺していたからだ。
-The reason why I hadn't felt any pain was because my entire body was numb.
+Because my entire body had been paralyzed, I hadn't felt it before.
 }
 [sha:0dba37d991dddb462ed5cf11cfa30dfd06679473]{
 -- Page 90, Offset 31929.
 -- こうやって少しずつ神経を取り戻していけば、同時に痛みも取り戻していくのは当然だった。
-As my nerves slowly regain function, it's only natural that the pain would return as well.
+It makes perfect sense that as my nerves begin to slowly regain their function,
+ that sense of pain will come back as well.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:131]: my sense of pain

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 3]
```diff
@@ -132,27 +138,27 @@ As my nerves slowly regain function, it's only natural that the pain would retur
...
 -- Page 92, Offset 31933.
 -- しかし、こうして体の自由を取り戻さないと部屋から出られない。
-But if I don't recover my body's freedom, I won't be able to leave this room.
+But if I don't regain control of my body, I won't be able to leave this room.
 }
 [sha:c4f4ce2e8d10178d47015eb789a7c976922b653c]{
 -- Page 92, Offset 31934.
 -- 部屋から出て、街に出て、学校に行って、
-I won't be able go out to the city, into the school,
+I won't be able to get of here, head for the city, or make it the school―――
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:156]: to the school

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 4]
```diff
@@ -387,44 +408,52 @@ Senpai sits on the chair that Akiha was sitting on earlier.
...
-"Then I'll ask. That guy from earlier was Roa, wasn't he?"
+"Alright, here goes. That guy from earlier was Roa, wasn't he?"
 }
 [sha:363ea0f2617501c82a4425bd6f1f5597b384c0c1]{
 -- Page 111, Offset 31986.
 -- 「……はい。アレが今代のロアの転生体です。七年前に遠野くんの命を略奪した遠野シキ。……そのあたりの事情は秋葉さんに聞きました？」
-// cut reincarnation since implied
-"...That's right. He's the Roa of this period. The one who robbed you of your life seven years ago, Tohno SHIKI... have you heard the story from Akiha-san?"
+"...That's right. He's the host for the current reincarnation of Roa.
+ The one who robbed you of your life seven years ago, Tohno SHIKI...
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:435]: comma before ago makes it sound like she's addressing Tohno SHIKI.
> ago: Tohno SHIKI

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 5]
```diff
@@ -479,72 +509,81 @@ In that case―――there's still a chance to catch her.
...
 -- ロアは転生する吸血鬼。その時の姿や性格、嗜好性が変化するのは当然です」
-Roa is a reincarnating vampire. So it's only natural that his appearance, character, and even his tastes would change."
+Roa is a reincarnating vampire. It's only natural that his appearance, character, and even his tastes would change."
 }
 [sha:2a81be82906961c56d3b4f57182dde176aef28be]{
 -- Page 119, Offset 32004.
 -- 「いいですか遠野くん。ロアが転生先に選んだ肉体は、ロアとは別の人格・人間として育ちます。知性が十分に育たなければロアは己を発揮できませんから。
-"Listen to me, Tohno-kun. The body that Roa chooses before reincarnating is raised as a different being, with a different personality. If it doesn't develop enough intelligence, then Roa will not be able to manifest.
+"Listen to me, Tohno-kun.
+ The body that Roa chooses to reincarnate into is raised as a human, with their own personality and everything.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:535]: drop comma after human

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 6]
```diff
@@ -869,22 +927,23 @@ What do you mean she's dying?
...
 -- Page 150, Offset 32080.
 -- けれどアルクェイドは二度と人の血は吸わないでしょう。
-But Arcueid wouldn't suck another human's blood ever again, right?
+But Arcueid isn't going to suck anyone's blood again, right?
 }
 [sha:7c1c689db5322204706d401db362b2b844847ffb]{
 -- Page 150, Offset 32081.
 -- だからロアを追いかけるかぎり、彼女は自分から死に近づいているんです」
-That's why as long as she's going after Roa, she will always be drawing closer to death."
+That's why she will continue to drive herself closer and closer to death the longer she chases after Roa."
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:946]: we got some vampiric calculus here

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 7]
```diff
@@ -1049,17 +1113,20 @@ If it's about Roa, he will be taken care of in a couple of days."
 [sha:18b80606a95832077d7a6e7adfd3025076bcf61b]{
 -- Page 164, Offset 32114.
 -- 「ロアの転生体が特定できましたから、法王庁……えっと、わたしたちの本拠地ですけど、そこに要請が通ります。
-"Since we've identified Roa's reincarnate, we'll finally be able to request aid from the Curia... uhhh, our headquarters.
+"Now that we've identified Roa's reincarnation, we can finally request assistance from the Vatican... uhh, our headquarters.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1116]: I'm not catholic so I'm going off of Wikipedia descriptions here, but the Curia appears to be a specifically the authoritative body in the church. I'm guessing you're using "Vatican" here like "White House" or "Pharaoh" but even so, I think that Curia is right.
> comprises the administrative institutions of the [Holy See](https://en.wikipedia.org/wiki/Holy_See)[[note 1]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-1) and the central body through which the affairs of the [Catholic Church](https://en.wikipedia.org/wiki/Catholic_Church) are conducted. It acts in the [pope](https://en.wikipedia.org/wiki/Pope)'s name and with his authority for the good and for the service of the [particular churches](https://en.wikipedia.org/wiki/Particular_church) and provides the central organization for the church to advance its objectives.[[1]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-2)[[2]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-3)[[3]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-4)[[note 2]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-6)

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 8]
```diff
@@ -1299,17 +1376,21 @@ After all, her smile is a truly beautiful sight.
...
-I have to make her even happier for all the times she hadn't been saved. It would be wrong for things to stay like this.
+// Mild liberty 'un-saved' -> alone for both fragments
+I have to make her even happier to make up for all the time she's spent on her own.
+ It would be unforgivable not to balance out that loneliness.
 }
 [sha:621584ec425a2d4c020488b67ffc8d5914c6431d]{
 -- Page 182, Offset 32166.
 -- だいたい、あいつが独りでなくなる方法なんて簡単すぎる。そんな事で悩んでるとか、馬鹿みたいだ」
-Besides, it would be too simple for her to go die alone. It's stupid for her to worry about things like that."
+// Is Nasu trying to tell his audience to go outside and make some friends
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1391]: he can't make me

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 9]
```diff
@@ -1639,6 +1726,7 @@ No, of course not.
 [sha:db47cc9d6e32e9e452ce99a8a05907fb771c16fd]{
 -- Page 208, Offset 32233.
 -- 「知るか。言う事をきかないなら、ここでメチャクチャに暴れてやる」
+// Real classy
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1729]: classier than in the og i guess

**Review by Avuxo — APPROVED** [review]
what a miserable file.

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 13/13_00_ARC13_1B.txt:109]: Ah, I got that confused with synesthesia

**Pull request merged**